méli-mélo/deprecated/style.css

```diff
@@ -9,3 +9,31 @@
 .well.well-bold strong {
 	font-weight: 400;
 }
+
+/* Background image heading: deprecated as of v16.0.0 */
+.bg-img-hdng {
+	background-image: url("https://www.canada.ca/content/dam/servcan/x-blue-line.png");
+	background-position: right bottom;
+	background-repeat: no-repeat;
+	min-height: 60px;
+	padding-bottom: 35px;
+	padding-right: 80px;
+}
+
+/* Light warning alert icon: deprecated as of v16.0.0 */
+.provisional.icon-warning-light, .provisional .icon-warning-light {
+	color: #DF7200;
+}
+
+/* Google+ in Follow us component: deprecated as of v16.0.0 */
+.googleplus {
+	background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAC4AAAAuCAYAAABXuSs3AAAAAXNSR0IArs4c6QAAA9lJREFUaAXtmX9IU1EUx882tznnr0ynGREpFvRfkWZ/FCRSRGQWmEgU/VF/FNEPqD8LwiAoIUihfyMoSyywoP6oDCEtqUj6IWRIhKVu/pq2qU+39zrnuvt82x6Mt+3pgl04vHPPu+/cz/t6d3bvBPhPm0GFWy2mMmzJQ5JyRiUk+aaAka+8p3xmqX0CJvMHjL1AioLC9KayvCrPZm00GYyFiviyu35JHHTNzp3e/uLtU4TxEZAxQEXqmhMRmvhISEeqpYkY0dhKUIJbEk1pguYtwGbBfjg4H5TAV1Vw5XpPVHZiDFKcQFkgUYkDXDIjX+MJzhuOlwQP10TfSFJxffUNz55UPFwTfSMxfekY7HbIPlgH9i1bwZyP+zKssvPOIfB+6AZ36z2Qpqd1o48aPG1zGeSePAdmR0EQnHVtEZBl7KgAZ1MDCD0fg+7HqxMVuG1TKRRcvgYGw8IX2dyIC2a+9zIm24aNYMlzgOT3g2/wd7w4w/JoBjek2SHv1HkGLfp8MNZ8ByYfPwCDKLLkktEI6bv3wUx3J4jjo2ETxiugGTz7QK28PCYeNcNU6/2gTQ69gPd5W0S+4rZ2NqZ/f0XEsWoDNJdDe2k5yyPiB8/dclct55LENCtuLljNwISfP/AQRcfAxcZVXIwseJ7ODnBev8I6oWN4X6vymhWXJH7YlneYoZzhfROdwePbNCvuc/4B07oSsBaV4AkQj4Dz8zLRcGOD7JuysiDv6AnW942PyfG+qp3MX//kNbvyvilQoeSBERzNinvedbGURpsNcuqOBaX3vnwG3ExmOmUtNKH3C3fx4GtgxgOhfR6PdNWs+GRbC2RU7AJL/irIrj4EEpbEiYf4IcW6zVvm3mpYUXOYdYWBX+Dp6uC35Gu0SvMEmsGlmRlw3boBhfUNYMS1m1N7BDIr98Bsfx+AKOESKpbLpYilceT2zaCX4hNrXRr8OX7VDE4PCl97YPDSBXCcuciUT1mZC+loyuZzT4ALoYVvn5XhuPlRgdPsBD9w9jhkVtWAvWwbWAvXBDZZw+D99B4m8ctJ+jsVN9DQRFGDs0S4bKZwfZPxIglYLvkeJnSyePZjA1eQyFVdY1lTpNDkai6HmrLrODgJrqO4qqmTiqvKomMwqbiO4qqmTiquKouOQaXioiD6XTrOFVPqANvCTwmYiYPTVsPXPjR6VfD7R2KaQYeHienV8Fg9MaKxbRHfYtALpKPlo+WgpaLxe+guayPQWbRxNCeaB03kmyy6KaC5KYhG5y7+10B3WRvxzKHRHpkYgxTHPgOlf4ASNL1QIilOS4Tg6WTO1vk/yrUG/vk8ZeYAAAAASUVORK5CYII=");
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-size: 35px 35px;
+	display: inline;
+	min-height: 27px;
+	min-width: 27px;
+	position: relative;
+	vertical-align: text-bottom;
+}
```

Comment:
We should have a copy of all the depreceated css in gc-follow-us/_base.scss here